router/core/src/main/scala/io/buoyant/router/LabelerConfig.scala

```diff
@@ -0,0 +1,68 @@
+package io.buoyant.router
+
```

Comment:
Can this file be moved into linkerd/core?  Since it's a config file it probably belongs in linkerd rather than router.  This used to be part of AddForwardedHeaderConfig which is also moving to linkerd/core so I think it makes sense for this to move into linkerd/core as well.

Reply:
Just made the change and looks like it is possible. 👍